VirtoCommerce.Platform.Web/Modularity/ManifestModuleCatalog.cs

```diff
@@ -170,25 +173,59 @@ public override void Validate()
             return result;
         }
 
+        private static void CopyLocalizationAssemblies(string sourceDirectoryPath, string targetDirectoryPath)
+        {
+            if (Directory.Exists(sourceDirectoryPath))
+            {
+                foreach (var sourceFilePath in Directory.EnumerateFiles(sourceDirectoryPath, "*.resources.dll", SearchOption.AllDirectories))
+                {
+                    var separator = Path.DirectorySeparatorChar;
+                    var directory = Path.GetDirectoryName(sourceFilePath);
+
+                    var languageCode = directory?.Split(separator).Last();
```

Comment:
Here a bug could be introduced. 
I think we need to ensure we have only one directory in source resource DLL path, otherwise path localization file from path like "/A/B/en-US/" will be copied (probably should not). And then fullSourceDirectoryPath will be wrong.
You could use first level subdirectory search
```var files = 
    Directory.EnumerateDirectories(root).SelectMany(
        directory => Directory.EnumerateFiles(directory, "*.resources.dll"));
```
Seems `languageCode` and manual path construction will not be needed at all.

Reply:
done